### DISCUSSION TITLE
What is "minting IRIs" and how do you do it?

### DISCUSSION THREAD

**Discussion by JisooSeo** [JisooSeo/101-WG-Discussions#5] [Q&A] [answered]
What is "minting IRIs" and how do you do it?

**Comment by giacomodecolle** [comment, marked as answer]
Every resource on the internet has its own unique identifier. E.g. the URL of github that you type in your browser tab. Ontology classes and individuals and properties also have their own unique identifier, usually called an URI or IRI (can't remember the exact difference, Semantic Web for the Working Ontologist is a good resource). When you create an ontology, you also create ("mint") these unique identifiers. It is usually good to have a certain strategy behind them. More recently, the trend has been to have IRIs created like BFO:0000001, i.e. non-human readable IRIs, and then add the human-readable name through the rdfs:label. Remember that once you create your IRIs they are not going to be changed, and people will always refer to them. That's why you keep them agnostic and you change the labels. 
This is my understanding so far at least.

**Reply by JisooSeo** [reply]
Thank you so much for the comment! 😄
